apps/web/app/organizations/invitations/[invitationId]/accept/page.tsx

```diff
@@ -0,0 +1,127 @@
+"use client";
+
+import { useParams, useRouter } from "next/navigation";
+import { useEffect, useState } from "react";
+import { Button } from "@/components/ui/button";
+import {
+  Card,
+  CardContent,
+  CardDescription,
+  CardHeader,
+  CardTitle,
+} from "@/components/ui/card";
+import { Loader2 } from "lucide-react";
+import { useUser } from "@/hooks/useUser";
+import { handleInvitationAction } from "@/utils/actions/invitation";
+import { mutate } from "swr";
+import { setInvitationCookie, clearInvitationCookie } from "@/utils/cookies";
+
+export default function AcceptInvitationPage() {
+  const params = useParams();
+  const router = useRouter();
+  const { data: user, isLoading: userLoading } = useUser();
+  const [loading, setLoading] = useState(true);
+  const [error, setError] = useState<string | null>(null);
+  const [success, setSuccess] = useState<{ organizationName: string } | null>(
+    null,
+  );
+  const [hasProcessed, setHasProcessed] = useState(false);
+
+  const invitationId = params.invitationId as string;
+
+  // Revalidate user data to ensure we have fresh data
+  useEffect(() => {
+    mutate("/api/user/me");
```

Comment:
const { data: user, isLoading: userLoading, mutatate } = useUser();

and then just call mutate without the need for the key here